packages/styled-base/tests/flow.js

```diff
@@ -0,0 +1,8 @@
+// @flow
```

Comment:
could you rename this directory to smth like `flow-tests`? `tests` is too generic and we probably want to keep flow-related type tests in a separate directory - so its name should be intuitive